### PR TITLE
fix(inline-input): update story and knobs - FE-2786

### PR DESCRIPTION
**Pull request by damianOfKyan**
### Proposed behaviour
Story knob should accept special characters such as <&>

### Current behaviour
Text Knob in story is not HTML escaped

### Checklist

- [x] Carbon implementation and Design System documentation are congruent
- [x] Release notes (Conventional Commits) 
<del>- [ ] Unit tests
- [x] Cypress automation tests
- [x] Storybook added or updated
- [x] Theme support
- [x] Typescript `d.ts` file added or updated

### Testing instructions
http://localhost:9001/?path=/story/design-system-inline-inputs--default-story

### CODE REVIEW

File: cypress/features/regression/buttonToggle.feature [hunk 1]
```diff
@@ -6,12 +6,14 @@ Feature: Button Toggle component
...
   @positive
   Scenario Outline: Set Button Toggle childen to <label>
-    When I set children to "<label>"
-    Then Button Toggle label on preview is "<label>"
+    When I set children to <label> word
+    Then Button Toggle label on preview is <label>
     Examples:
-      | label                   |
-      | mp150ú¿¡üßä             |
+      | label                        |
```
Comment (DlgSHi) [inline cypress/features/regression/buttonToggle.feature:12]: spacing

File: cypress/features/regression/stepSequenceItem.feature [hunk 2]
```diff
@@ -28,35 +30,40 @@ Feature: Step Sequence Item component
...
-    When I set hiddenCompleteLabel to "<hiddenCompleteLabel>"
+    When I set hiddenCompleteLabel to <hiddenCompleteLabel> word
       And I select status to "complete"
-    Then hidden label is set to "<hiddenCompleteLabel>"
+    Then hidden label is set to <hiddenCompleteLabel>
     Examples:
-      | hiddenCompleteLabel |
-      | mp150ú¿¡üßä         |
+      | hiddenCompleteLabel     |
+      | mp150ú¿¡üßä             |
```
Comment (DlgSHi) [inline cypress/features/regression/stepSequenceItem.feature:38]: will we check here also special characters ?

File: cypress/features/regression/stepSequenceItem.feature [hunk 3]
```diff
@@ -28,35 +30,40 @@ Feature: Step Sequence Item component
...
   Scenario Outline: I set hiddenCurrentLabel to <hiddenCurrentLabel>
-    When I set hiddenCurrentLabel to "<hiddenCurrentLabel>"
+    When I set hiddenCurrentLabel to <hiddenCurrentLabel> word
       And I select status to "current"
-    Then hidden label is set to "<hiddenCurrentLabel>"
+    Then hidden label is set to <hiddenCurrentLabel>
     Examples:
-      | hiddenCurrentLabel |
-      | mp150ú¿¡üßä        |
+      | hiddenCurrentLabel      |
```
Comment (DlgSHi) [inline cypress/features/regression/stepSequenceItem.feature:46]: will we check here also special characters ?

File: cypress/features/regression/stepSequenceItem.feature [hunk 4]
```diff
@@ -28,35 +30,40 @@ Feature: Step Sequence Item component
...
   Scenario Outline: I set hiddenCompleteLabel to <hiddenCompleteLabel>
-    When I set hiddenCompleteLabel to "<hiddenCompleteLabel>"
+    When I set hiddenCompleteLabel to <hiddenCompleteLabel> word
       And I select status to "complete"
-    Then hidden label is set to "<hiddenCompleteLabel>"
+    Then hidden label is set to <hiddenCompleteLabel>
     Examples:
-      | hiddenCompleteLabel |
-      | mp150ú¿¡üßä         |
+      | hiddenCompleteLabel     |
```
Comment (DlgSHi) [inline cypress/features/regression/stepSequenceItem.feature:37]: spacing

File: cypress/features/regression/stepSequenceItem.feature (hunk 3 above)
Comment (DlgSHi) [inline cypress/features/regression/stepSequenceItem.feature:46]: spacing

File: cypress/features/regression/tableAjax.feature [hunk 5]
```diff
@@ -26,7 +26,7 @@ Feature: Table Ajax component
       | -1                      |
```
Comment (DlgSHi) [inline cypress/features/regression/tableAjax.feature:26]: formatting

File: cypress/features/regression/tableAjax.feature [hunk 6]
```diff
@@ -53,4 +53,4 @@ Feature: Table Ajax component
     Examples:
       | getCustomHeaders        |
```
Comment (DlgSHi) [inline cypress/features/regression/tableAjax.feature:54]: formatting

File: cypress/features/regression/buttonToggle.feature (hunk 1 above)
Comment (KatarzynaQA) [reply cypress/features/regression/buttonToggle.feature]: fixed

File: cypress/features/regression/stepSequenceItem.feature (hunk 2 above)
Comment (KatarzynaQA) [reply cypress/features/regression/stepSequenceItem.feature:38]: here it's not require

File: cypress/features/regression/stepSequenceItem.feature (hunk 3 above)
Comment (KatarzynaQA) [reply cypress/features/regression/stepSequenceItem.feature:46]: here it's not require

File: cypress/features/regression/stepSequenceItem.feature (hunk 4 above)
Comment (KatarzynaQA) [reply cypress/features/regression/stepSequenceItem.feature:37]: fixed

File: cypress/features/regression/stepSequenceItem.feature (hunk 3 above)
Comment (KatarzynaQA) [reply cypress/features/regression/stepSequenceItem.feature:46]: fixed

File: cypress/features/regression/tableAjax.feature (hunk 5 above)
Comment (KatarzynaQA) [reply cypress/features/regression/tableAjax.feature]: fixed

File: cypress/features/regression/tableAjax.feature (hunk 6 above)
Comment (KatarzynaQA) [reply cypress/features/regression/tableAjax.feature]: fixed

**Pull request merged**